.github/workflows/ci.yaml

```diff
@@ -81,11 +81,11 @@ jobs:
             ccache-${{ env.CACHE_PREFIX }}-${{ github.sha }}
             ccache-${{ env.CACHE_PREFIX }}
       - name: industrial_ci
-        uses: tylerjw/industrial_ci@clang-tidy-modified-filter
+        uses: ros-industrial/industrial_ci@master
         env: ${{ matrix.env }}
-      - name: upload test artifacts (on failure)
+      - name: upload test artifacts
         uses: actions/upload-artifact@v2
-        if: failure()
+        if: always()
```

Comment:
Why do you want to upload test results always now?

Reply:
That's left over from me trying to use the that report action.  I'll revert that.

Reply:
fixed